lib/recurly/base_client.php

```diff
@@ -20,12 +24,23 @@ abstract class BaseClient
 
     /**
      * Constructor
-     * 
+     *
      * @param string $api_key The API key to use when making requests
+     * @param string $options initialize options
+     *
+     * In addition to the options managed by BaseClient, it accepts the following options:
+     *  - "region" to define the Data Center connection - defaults to "us";
      */
-    public function __construct(string $api_key, LoggerInterface $logger = null)
+    public function __construct(string $api_key, LoggerInterface $logger = null, array $options = [])
```

Comment:
I regret making the `$logger` it's own parameter instead of using `$options` when it was introduced. This constructor signature is not very friendly.